File: Lec-1/Assignment/Java/Input_from_user_Array.java
package Assignment.Java;

import java.util.Scanner;

public class Input_from_user_Array
{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the size of the array: ");
        int size=sc.nextInt();
        int[] array=new int[size];
        System.out.print("Insert the Elements in the array: ");
        for(int i=0;i<size;i++)
        {
            array[i]=sc.nextInt();
        }
        for(int i=0;i<size;i++)
        {
            System.out.print(array[i]+" ");
        }
        sc.close();
    }


}

File: Lec-1/Assignment/Cpp/Input_from_user_array.cpp
#include <iostream>
using namespace std;

int main()
{
    int size;
    cout<<"Enter the size of the array: ";
    cin>>size;

    int arr[size];
    cout<<"Insert the elements in the array: ";
    for(int i=0;i<size;i++)
    {
        cin>>arr[i];
    }
    for(int i=0;i<size;i++)
    {
        cout<<arr[i]<<" ";
    }
}